README.md

```diff
@@ -57,7 +57,7 @@ Almost 90% of the app that I have developed has an Image upload feature. Along w
 	```
 
     ```groovy
-   implementation 'com.github.dhaval2404:imagepicker:2.1'
+   implementation 'com.github.aazeemcse:ImagePicker:v2.1.1'
```

Comment:
Why should I approve this?

Reply:
> Why should I approve this?
Hi don't approve the changes in readme. as I have to use the new changes in my project so I have created my own repo and published to use. if you can pull the free style cropping changes I will use your repo aging.